src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/Utils.java

```diff
@@ -75,8 +76,8 @@ public static VarHandle fixUpVarHandle(VarHandle handle) {
         return MemoryHandles.filterCoordinates(handle, 0, ADDRESS_FILTER);
```

Comment:
The above comment needs updating to refer to `MemorySegmentProxy` and `MemorySegment`.